### ISSUE TITLE
Is loss computed on 12 x 12 output prediction or after Deconvolution/Upsampling?

### ISSUE THREAD

**Issue by anilbatra2185** [richzhang/splitbrainauto#6]
As per the architecture, the CL/Reg loss is calculated after down sampling ground truth to 12 x 12  or upsampling 12 x 12 prediction to 180 x 180?

![sbae](https://user-images.githubusercontent.com/7984207/38706613-6ba2e89c-3ecb-11e8-8f17-06770441b789.png)

Also, I am not able to understand the reason for absorbing BatchNorm layer into conv layer after pretraining, is there any effect of this? 

Thanks

**Comment by fungtion** [comment]
I have the same question, it seems that the author downsized the groundtruth to 12x12.

**Comment by richzhang** [comment]
Hi, I downsampled ground truth.
